api/types/entrySchema.go

```diff
@@ -12,6 +12,8 @@ import (
 
 // EntrySchema describes an entry's schema, which
 // is what's returned by the /fs/schema endpoint.
+//
+// swagger:response
```

Comment:
I've been having trouble installing swagger on my machine and getting it to work. Also, not sure how to document this since the type is not a 1:1 mapping with the response (the latter is a `<type_id> => <schema>` map w/ first key being entry's type ID). So could use some help with this part.

Reply:
I'll try. Go 1.12 is still only part-way through the Go Modules migration, so I keep the wash project in GOPATH and run swagger without go modules (meaning I have to separately `go get` all the dependencies). Hopefully 1.13 will fix this.

Reply:
I'll plan to update the swagger docs as a separate PR.

Reply:
Ok, sounds good.